tests/everest/test_res_initialization.py

```diff
@@ -23,7 +22,9 @@
     SlurmQueueOptions,
     TorqueQueueOptions,
 )
-from everest.config import EverestConfig, EverestValidationError
+from everest.config import (
+    EverestConfig,
+)
```

Comment:
Any particular reason why we format like this (with the comma)?

Reply:
Ruff auto-formatting always leaves comma's. I usually remove always the last comma, ruff will put it back only when needed.